README.md

```diff
@@ -12,24 +9,19 @@ cd jdb_to_nwb
 pip install -e .
```

Comment:
Have separate installation instructions for released version (`pip install jdb-to-nwb`) vs development version?

Reply:
I was debating this -  I think for now I prefer to not encourage pip installing because we reply on a jupyter notebook `resources/Get_Hex_Centroids` to click the centroids from the video. And I don't think it would be included with a pip install?

Reply:
Right, that notebook would not be available via pip install. So that makes sense.